ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -253,3 +253,33 @@ ${local.skip_on_weekends}
     action_group = var.action_group_ids
   }
 }
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "bulk_results_upload" {
+  name                = "${var.env}-bulk_results_upload"
+  description         = "${local.env_title} alert when bulk uploads "
```

Comment:
Is this description unfinished?

Reply:
yes

Reply:
lol the description leaves you wanting more